scripts/airdrop.js

```diff
@@ -0,0 +1,54 @@
+const fs = require("fs").promises
+
+const { signAndSend, transactionExistsOnSafeServer } = require("./utils/gnosis_safe_server_interactions")(web3, artifacts)
+const { buildTransferDataFromList } = require("./utils/trading_strategy_helpers")(web3, artifacts)
+const { promptUser } = require("./utils/user_interface_helpers")
+const { default_yargs } = require("./utils/default_yargs")
+const argv = default_yargs
+  .option("masterSafe", {
+    type: "string",
+    describe: "Address of Gnosis Safe owning the brackets",
+    demandOption: true,
+  })
+  .option("transferFile", {
+    type: "string",
+    describe: "file name (and path) to the list transfers",
+    demandOption: true,
+  })
+  .option("verify", {
+    type: "boolean",
+    default: false,
+    describe: "Do not actually send transactions, just simulate their submission",
+  })
+  .option("useWei", {
+    type: "string",
+    describe: "file name (and path) to the list transfers",
+    demandOption: false,
+  }).argv
+
+module.exports = async (callback) => {
+  try {
+    const GnosisSafe = artifacts.require("GnosisSafe")
+    const masterSafe = await GnosisSafe.at(argv.masterSafe)
+
+    const transfers = JSON.parse(await fs.readFile(argv.transferFile, "utf8"))
+
+    console.log("Preparing transaction data...")
+    const transaction = await buildTransferDataFromList(masterSafe.address, transfers, argv.useWei, false, true)
+
+    if (!argv.verify) {
+      const answer = await promptUser("Are you sure you want to send this transaction to the EVM? [yN] ")
+      if (answer == "y" || answer.toLowerCase() == "yes") {
```

Comment:
I would also consider lowercasing the answer when comparing against `y`

```suggestion
      if (answer.toLowerCase() == "y" || answer.toLowerCase() == "yes") {
```


Of if you want to be fancy, use a switch :)

```ts
switch (answer.toLowerCase()) {
  case `y`:
  case `yes`:
  case `yeah`:
  case `yup`:
  case `yep`:
  case `hell yeah`:
  case `sure`:
  case `why not, eh?`:
    ...
  default:
    //no
}

Reply:
I like the suggestion, but it actually makes me want to restrict it now to only lower case `y` because that is what is suggested in the prompt (i.e. `[yN]`)